packages/liveblocks-react-ui/src/components/internal/AiChatComposer.tsx

```diff
@@ -168,6 +177,7 @@ export const AiChatComposer = forwardRef<HTMLFormElement, AiChatComposerProps>(
           {
             stream,
             copilotId,
+            knowledge: localKnowledge,
```

Comment:
With https://github.com/liveblocks/liveblocks/pull/2534, I would love for us to use the public `useSendAiMessage` API in the composer to allow future primitives users to do the same things, but this option seems niche enough that we could keep it as a private option in `useSendAiMessage`.